react-d3-chart/src/utils/index.ts

```diff
@@ -17,44 +18,34 @@ export const downSample = (data: LineProps[], maxPoints: number) => {
 };
 
 /**
- * Will return the smallest and largest values over any line for the given dimension
+ * Will return the smallest and largest values over any line for the given
+ * dimension
  * @param data Data points
  * @param dimension Calculate bounds for this dimension
- * @returns The calculated lower and upper bounds for the given dimension and data points
+ * @returns The calculated lower and upper bounds for the given dimension and
+ *     data points
  */
-const getDomainHelper = (
-  data: CommonProps['data'],
-  dimension: 'x' | 'y'
-): Domains['xDomain'] => {
-  return [
-    Math.floor(
-      data.reduce(
-        (minSoFar, currLine) =>
-          Math.min(
-            minSoFar,
-            currLine.reduce(
-              (minVal, currCoord) => Math.min(minVal, currCoord[dimension]),
-              0
-            )
-          ),
-        0
-      )
-    ),
-    Math.round(
-      data.reduce(
-        (maxSoFar, currLine) =>
-          Math.max(
-            maxSoFar,
-            currLine.reduce(
-              (minVal, currCoord) => Math.max(minVal, currCoord[dimension]),
-              0
-            )
-          ),
-        0
-      )
-    ),
-  ];
-};
+const getDomainHelper =
+    (data: CommonProps['data'], dimension: 'x'|'y'): Domains['xDomain'] => {
+      return [
+        Math.floor(data.reduce(
+            (minSoFar, currLine) => Math.min(
+                minSoFar,
+                currLine.reduce(
+                    (minVal, currCoord) =>
+                        Math.min(minVal, currCoord[dimension]),
```

Comment:
I've been told I'm a separate review not to use abbreviations. Up to you if you think that's unnecessary.

Reply:
This is not from Craig - I think this line here just got reformatted by my IDE - We can discuss and potentially change in a sep PR